Source/WebKitLegacy/WebCoreSupport/WebResourceLoadScheduler.h

```diff
@@ -83,8 +83,10 @@ class WebResourceLoadScheduler final : public WebCore::LoaderStrategy {
     bool isSuspendingPendingRequests() const { return !!m_suspendPendingRequestsCount; }
     void isResourceLoadFinished(WebCore::CachedResource&, CompletionHandler<void(bool)>&&) final;
 
-    class HostInformation : public CanMakeWeakPtr<HostInformation>, public CanMakeCheckedPtr<HostInformation> {
-        WTF_MAKE_NONCOPYABLE(HostInformation); WTF_MAKE_FAST_ALLOCATED;
+    class HostInformation final : public CanMakeWeakPtr<HostInformation>, public CanMakeCheckedPtr<HostInformation> {
+        WTF_MAKE_NONCOPYABLE(HostInformation);
+        WTF_MAKE_FAST_ALLOCATED;
```

Comment:
Shouldn't we have `WTF_MAKE_FAST_ALLOCATED` on CanMakeCheckedPtr instead so we don't have to specify it for every subclass?

Reply:
Some classes use WTF_MAKE_ISO_ALLOCATED or WTF_MAKE_FAST_ALLOCATED_WITH_HEAP_IDENTIFIER.

Other subclasses have an incompatible definition of memory allocation. For example, GlyphBuffer is stack-allocated; and WebProcessPool uses API::ObjectImpl's Objective-C allocator. 

I was hoping that putting the allocation specification in each subclass would help clarify the requirements. 

I see now that RefCounted specifies WTF_MAKE_FAST_ALLOCATED though. I guess it provides a good default that way. But CanMakeWeakPtr doesn't. 

I guess I'm not sure. Should I add it?